third_party/render_html.go

```diff
@@ -1,15 +1,20 @@
 package devrel_tutorial
 
 import (
-	"strings"
 	"text/template"
 )
 
 func init() {
-	funcMap := template.FuncMap{
-		"safeHTML": func(s string) htmlTemplate.HTML {
-			return template.HTML(s)
-		},
-	}
-	html = template.Must(template.New("master").Funcs(funcMap).ParseGlob(htmlTmplsDir))
+	html = template.Must(template.New("master").ParseGlob(htmlTmplsDir))
```

Comment:
Is this supposed to be undoing stuff that was in PR 2? If so should we just remove it from PR 2?

Reply:
I found a better way to do this, so it's not an undo. But the change definitely needs to be on Part 2. Will clean that one in general a bit.